upgrade/php/ps_900_update_configuration.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+/**
+ * Init new configuration values
+ */
+function ps_900_update_configuration()
+{
+    Configuration::updateValue('PS_FEATURE_VALUES_ORDER', 'name');
+}
```

Comment:
This code does not seem called by the 9.0.0.sql script.

If you want to add configuration values to the shop, you may refer to this other SQL query:
https://github.com/PrestaShop/autoupgrade/blob/e1658640a47966e6cceaf20a202186886d74fc6f/upgrade/sql/9.0.0.sql#L58-L60